src/alerts/Alerts.ts

```diff
@@ -163,7 +183,11 @@ const AlertsIndicator = ({ style }) => {
     ),
 
     // Have screen readers announce alerts.
-    _.map(({ id, title, message }) => div({ key: id, role: 'alert', className: 'sr-only' }, [div([title]), !!message && div([message])]), alerts),
+    _.map(
+      ({ id, title, message }) =>
+        div({ key: id, role: 'alert', className: 'sr-only' }, [div([title]), !!message && div([message])]),
+      alerts
+    ),
   ]);
 };
 
```

Comment:
exporting as default is something we are trying to move away from... would it be too disruptive to change the main export to be named instead?

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4400/commits/574ab686e3106dfd38cf417ae5107eba364325e1.